docs/available_commands.md

```diff
@@ -207,7 +207,13 @@ Currently, two formats are supported: The [warnings-ng-plugin](https://github.co
 
 ## build-public
 
-This command creates a reduced version of an SBOM fit for publication. It:
+This command creates a reduced version of an SBOM fit for publication.
+
+If no JSON schema is provided the command:
+
+* removes any *property* (i.e., item in the `properties` array of a component) whose name starts with `internal:` from all components.
```

Comment:
Not  `name` but `namespace`. I would also add the information that this is done according to https://github.com/CycloneDX/cyclonedx-property-taxonomy#registered-top-level-namespaces

Reply:
Changed to "namespace starts with `internal:` from all components, according to [cyclonedx-property-taxonomy](https://github.com/CycloneDX/cyclonedx-property-taxonomy#registered-top-level-namespaces)."